test/e2e/metering_manual_install_test.go

```diff
@@ -249,7 +249,7 @@ func createMySQLDatabase(ctx *deployframework.DeployerCtx) error {
 		"oc",
 		"-n", mysqlNamespace,
 		"new-app",
-		"--image-stream", "mysql:5.7",
+		"--image-stream", "mariadb:10.2",
```

Comment:
Just a side note too, I would prefer to not outright disable this test as it's nice to have and could provide a way to quickly test whether or not the Mysql8 + metering BZ works or not, depending on how we prioritize that work.

Reply:
@timflannagan1 perhaps we should put a more about hack to delete the pod to get the hive-metastore to start creation `delete pod -l app=hive`